src/lib/import_export/csv_meta.cpp

```diff
@@ -0,0 +1,86 @@
+#include "csv_meta.hpp"
+
+#include <fstream>
+#include <string>
+#include <vector>
+
+#include "utils/assert.hpp"
+
+namespace opossum {
+
+CsvMeta process_csv_meta_file(const std::string& filename) {
+  std::ifstream metafile{filename};
+  Assert(metafile.good(), "Meta file does not exist: " + filename);
+  nlohmann::json meta_json;
+  metafile >> meta_json;
+  CsvMeta meta = meta_json;
+  return meta;
```

Comment:
maybe condense the last two lines to: `return static_cast<CsvMeta>(meta_json)` ?

Reply:
yep that works.